src/main/java/uk/gov/digital/ho/hocs/workflow/api/FormResource.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.digital.ho.hocs.workflow.api;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.digital.ho.hocs.workflow.api.dto.GetFormResponse;
+
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@Slf4j
```

Comment:
This isn't required.

Reply:
```suggestion
```

Reply:
Why didn't I think of that - I was putting off pushing a change 😆